src/plugins/data_source_management/public/plugin.ts

```diff
@@ -34,6 +39,7 @@ export interface DataSourceManagementPluginSetup {
   ui: {
     DataSourceSelector: React.ComponentType<DataSourceSelectorProps>;
     getDataSourceMenu: <T>() => React.ComponentType<DataSourceMenuProps<T>>;
+    dataSourceURLKey: string;
```

Comment:
Shall we use `typeof DATA_SOURCE_URL_KEY` here?

Reply:
@huyaboo +1, can we use type instead of exposing it like this?